components/card.css

```diff
@@ -0,0 +1,203 @@
+.hz--card-container {
+	display: flex;
+	background-color: var(--teritiary-color);
+	width: 600px;
```

Comment:
you can use relative units like rems instead of px

Reply:
fixed